ui/components/ui/nft-info/index.scss

```diff
@@ -0,0 +1,14 @@
+.nft-info {
+  &__content {
+    width: 100%;
+
+    &__view {
+      flex-grow: 1;
```

Comment:
I'm mobile at the moment, so this is just a question, do we not currently support flexGrow as a property of the Box component?

Reply:
Hi @brad-decker, flexGrow is currently not supported